packages/trackerless-network/test/integration/PlumTreePropagation.test.ts

```diff
@@ -110,7 +110,7 @@ describe('Propagation', () => {
             await until(() => totalReceived >= NUM_OF_NODES * i, 10000)
         }
         totalReceived = 0
-        const numberOfNewNodes = 32
+        const numberOfNewNodes = NUM_OF_NODES
```

Comment:
These are two different configurable values? I.e. should we have something like this:
```ts
const INITIAL_NODE_COUNT = 8
const NEW_NODE_COUNT = 8
```